### PR TITLE
Updated Packet Capture troubleshooting guide to include Selective Packet Capture

### PR DESCRIPTION
**Pull request by MichaelBaj**

### CODE REVIEW

File: docs/ts_packet_capture.md [hunk 1]
```diff
@@ -81,3 +83,179 @@ An example of showing a capture filter is below:
...
+```
+========= ==== =================== =================== ========== ==================== =================
+ Service   Id   Source              Destination         Protocol   Remaining Sessions   Active Sessions
+========= ==== =================== =================== ========== ==================== =================
+ west       2   0.0.0.0/0:0-65535   0.0.0.0/0:0-65535   tcp        unlimited                          0
+ west       5   0.0.0.0/0:0-65535   0.0.0.0/0:0-65535   icmp       unlimited                          1
+ east       1   0.0.0.0/0:0-65535   0.0.0.0/0:0-65535   icmp       unlimited                          0
+```
+
+If you specify the service and capture ID via arguments, you will see details of the sessions being captured.  In the above example you can see that service “west” has an active session on capture `5`.  The detailed view can be seen below, where it show session “1640858e-fe6a-44cd-b38a-7d479a68418” is actively being captured:
```
Comment (ptimmons) [inline docs/ts_packet_capture.md:234]: typo: show should be shows

File: docs/ts_packet_capture.md [hunk 2]
```diff
@@ -2,28 +2,30 @@
 title: PCLI Packet Capture
 ---
 
-In troubleshooting issues in a network, time is of the utmost importance.  With that, PCAPs are one of the most useful tools to debug traffic issues on a 128T Router(s) as well as wider networking issues.  Given those two factors, there’s often a need to quickly add or remove a capture filter without a need to make it persistent.
+PCAPs are one of the most useful tools to debug traffic issues on a 128T Router(s) as well as wider networking issues.  Given those two factors, there’s often a need to quickly add or remove a capture filter without a need to make it persistent.
```
Comment (ptimmons) [inline docs/ts_packet_capture.md:5]: ... as well as wider networking issues, and there's often a need to quickly add...

(There aren't two factors given, so the start of the second sentence is awkward.)

File: docs/ts_packet_capture.md [hunk 3]
```diff
@@ -81,3 +83,179 @@ An example of showing a capture filter is below:
...
+
+### Default Mode
+
+There are four locations within the 128T wherein capturing packets will provide full visibility into the behavior of a packet.
+1. Forward flow of a session arriving on the ingress interface (before any NATing has been applied)
+2. Forward flow of a session leaving the egress interface (post NAT with decrypted metadata)
+3. Reverse flow of a session arriving on the egress interface
+4. Reverse flow of a session leaving the ingress interface
+
+When creating a selective capture filter on the LAN interface, sessions will be tagged with an action that will capture all packets for the session at each of the four points listed above. Additionally, [metadata](concepts_metadata.md) will indicate to subsequent 128T nodes/routers to enable the packet capture for this session. Each 128T node will install capture filters in each of the four capture points for the same session. A [pcap](https://www.tcpdump.org/pcap.html) file will be created on each node, containing the name of the service captured.
```
Comment (ptimmons) [inline docs/ts_packet_capture.md:107]: You should probably capitalize PCAP throughout.

File: docs/ts_packet_capture.md (hunk 2 above)
Comment (MichaelBaj) [reply docs/ts_packet_capture.md:5]: I had reworked the introduction a number of times.  Clearly needs another revision. ;)

**Pull request merged**